tests/test_idpi/test_config.py

```diff
@@ -0,0 +1,15 @@
+# First-party
+import idpi.config
+
+
+def test_config():
+    with idpi.config.set_values(opt1=1):
+        assert idpi.config.get("opt1") == 1
+
+        with idpi.config.set_values(opt1=False, opt2=2):
+            assert not idpi.config.get("opt1")
```

Comment:
```suggestion
            assert idpi.config.get("opt1") is False
```
not evals to True for anything falsey such as an empty container